scripts/generate-stt-languages.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
```

Comment:
Please use the output in this PR instead of adding this script.
That will ensure we do not publish values that have not yet been released.
https://github.com/NabuCasa/support/pull/646